client.go

```diff
@@ -189,6 +189,15 @@ type ovndb struct {
 func NewClient(cfg *Config) (Client, error) {
 	db := cfg.Db
 	// db string should strictly be OVN_Northbound or OVN_Southbound
+	switch db {
+	case DBNB, DBSB:
+		break
+	case "":
+		db = DBNB
+	default:
+		return nil, fmt.Errorf("Valid db names are: %s and %s", DBNB, DBSB)
+	}
+
```

Comment:
The change doesn't seem necessary. Meanwhile, L201-206, the old validation, were not removed.

Reply:
why? its more readable now

Reply:
@hzhou8 :Yes removed l201-206. My bad in testing!

Reply:
> why? its more readable now

Well, it doesn't look more (or less) readable to me, but it is good if at least it is more readable to you :)